.golangci.yml

```diff
@@ -61,7 +61,8 @@ run:
   # can use regexp here: generated.*, regexp is applied on full path;
   # default value is empty list, but default dirs are skipped independently
   # from this option's value (see skip-dirs-use-default).
-  #skip-dirs:
+  skip-dirs:
+    - api/
```

Comment:
Because `disable-all` is true in `api/.golangci.yml` anyway, couldn't we continue to run the main linters on `api`, and then only enable `gomodguard` in `api`? That way there's less duplicated configuration

this is totally a nitpick though

Reply:
I think the best part of having different golangci.yml is that we can enable more linters or be more picky on what is accepted for the api vs flyctl.

the last bit was adding it to pre-commit at https://github.com/superfly/flyctl/pull/1922/commits/ae74695b130dbaed1cae46e7ce96587653acaacc

Reply:
it's not a strong argument just a preference, happy to revisit it if not useful

Reply:
@alichay I'm about to merge but don't want without having a confirmation you are fine with it.

Reply:
saw your approval! sorry